SayLess/__init__.py

```diff
@@ -403,7 +476,7 @@ def homepage():
             # settings link for the accessible file
             file_name = "https://sayless.blob.core.windows.net/images/" + profile.filename
             
-        return render_template('home.html', username=email_check.username, bio=bio,filename=file_name)
+        return render_template('home.html', username=email_check.username, bio=bio)
```

Comment:
What the point of filename being here if it is never used? It should be used initially when the websites loads because default should be the current user's profile.

Reply:
@moulid15

Reply:
File is load in the iframe only. before we had no iframe thats why we had to load it in the homepage.

Reply:
I am going to remove them since they are not being used.

Reply:
@Shazmaan

Reply:
Yepp just remove the stuff as it has no value there.